var/spack/repos/builtin/packages/fontconfig/package.py

```diff
@@ -35,6 +35,7 @@ class Fontconfig(AutotoolsPackage):
     version('2.11.1', 'e75e303b4f7756c2b16203a57ac87eba')
 
     depends_on('freetype')
+    depends_on('gperf', type='build')
```

Comment:
I'm confused by what's happening in the older versions.

Earlier versions also [seem to] use gperf, e.g. this section of the 2.12.1 `src/Makefile.in`:

```
906 fcobjshash.h: fcobjshash.gperf
907     $(AM_V_GEN) $(GPERF) -m 100 $< > $@.tmp && \
908     mv -f $@.tmp $@ || ( $(RM) $@.tmp && false )
```

but looking through `install -v` output from the unpatched tree I can't find any sign that that bit gets invoked.  I *suspect* that the `fcobjshash.h` included in the source is new enough that the rule is not invoked.

```
-rw-r--r-- 1 hartzelg users  1721 Jul 10  2016 fcobjshash.gperf
-rw-r--r-- 1 hartzelg users 14669 Jul 10  2016 fcobjshash.h
```

In the newer versions there's an additional bit in the `configure.ac` (and `configure`) that tries to determine the arg type of the gperf lookup function.  That uses `gperf` and falls back on `unsigned int` (which seems to fail on my machine).

That's a long way of saying: "Yes, I'll constrain it, we can always loosen it if the need arises."